zwave_js_server/model/node.py

```diff
@@ -234,6 +241,25 @@ def interview_attempts(self) -> Optional[int]:
         """Return the interview_attempts."""
         return self.data.get("interviewAttempts")
 
+    def get_command_class_values(
+        self, command_class: CommandClass, endpoint: int = None
+    ) -> Dict[str, Union[ConfigurationValue, Value]]:
+        """Return all values for a given command class."""
+        return {
+            value_id: value
+            for value_id, value in self.values.items()
+            if value.command_class == command_class
+            and (endpoint is None or value.endpoint == endpoint)
+        }
+
+    def get_configuration_values(
+        self, endpoint: int = None
+    ) -> Dict[str, ConfigurationValue]:
+        """Return all configuration values for a node."""
+        return self.get_command_class_values(
```

Comment:
I think we need to cast the return value type here.

Reply:
I have done that but I am still getting type errors as follows:
```
zwave_js_server/model/node.py:78: error: Incompatible types in assignment (expression has type "Value", target has type "ConfigurationValue")
zwave_js_server/model/node.py:386: error: Incompatible types in assignment (expression has type "Value", target has type "ConfigurationValue")
zwave_js_server/model/node.py:393: error: Incompatible types in assignment (expression has type "Value", target has type "ConfigurationValue")
```

One solution that works is if we create an `__init__` for `ConfigurationValue` that accepts a `Value` and creates a new object entirely. We can initialize each in the `get_configuration_values` method instead of trying to set the right class type during initialization of the node. Thoughts on this approach? It does eliminate the mypy errors but it feels wasteful to create a copy of the object.